lib/mapper.js

```diff
@@ -22,13 +52,52 @@ exports.track = function(track){
     objCase.del(props, 'revenue');
   }
 
-  return {
-    email: track.email(),
-    userId: track.userId(),
-    eventName: track.event(),
-    createdAt: unixTime(track.timestamp()),
-    dataFields: props
-  };
+  var eventName = track.event();
+  var cart = props["products"] || props["cart"];
```

Comment:
Re: event name case insensitivity, that's handled automatically:

https://github.com/segmentio/integration/blob/master/lib/wrap-methods.js#L46-L58
https://github.com/segmentio/analytics-events/blob/master/index.js#L7

:)